PubgApiTest/BatchProcessor.cs

```diff
@@ -22,14 +22,18 @@ public BatchProcessor(PubgApiClient pubgApi)
             _pubgApi = pubgApi;
         }
 
-        public void ProcessRecentMatchesForPlayer(string playerName)
+        public void ProcessRecentMatchesForPlayer(string playerName, int numberOfMatches = 5)
         {
             string rawResponse = _pubgApi.ExecuteGetRequest("https://api.pubg.com/shards/steam/players?filter[playerNames]=" + playerName, false);
 
             var playerInfo = JObject.Parse(rawResponse);
 
             var matchDatas = playerInfo.SelectToken("data[0].relationships.matches.data");
 
+            var playerAccountId = playerInfo.SelectToken("data[0].id").ToString();
+
+            var machesProcessed = 0;
+
             foreach (var matchData in matchDatas)
```

Comment:
Could simplify the changes by going "foreach (var matchData in matchDatas.Take(numberOfMatches))"